deps/captnlog/src/captainslog.h

```diff
@@ -71,16 +72,36 @@ extern "C" {
 #define captainslog_fatal(x, ...) ((void)0)
 #endif
 
+/**
+ * Logging system settings
+ *
+ * @param level Determines the max level of logging that will be carried out.
+ * @param filename Sets the filename to log to, if this is NULL or the file cannot be opened, file logging is disabled.
+ * @param console Does the logging system print to stderr?
+ * @param syslog Does the logging system log to either syslog or the windows system logs?
+ * @param print_time Does the logging system print the time on a log line?
+ * @param print_file Does the logging system print the file on a log line?
+ */
+typedef struct captains_settings_s
+{
+    int level;
+    const char *filename;
+    bool console;
+    bool syslog;
+    bool print_time;
+    bool print_file;
```

Comment:
`print_file` setting is new. To avoid complicated `captainslog_init` with many confusing arguments, I added new struct with parameters.